scalafix-tests/input/src/main/scala/test/DisableRegex.scala

```diff
@@ -0,0 +1,49 @@
+/*
+rules = Disable
+Disable.symbols = [
+  {
+    regex = "scala.collection.mutable.*"
+    message = "use immutable structures"
+  }
+]
+
+Disable.unlessInside = [
+  {
+    safeBlock = "test.DisableRegex.IO"
+    symbols = [
+      {
+        regex = {
+          includes = "java.io.*"
+          excludes = "java.io.InputStream"
+        }
+        message = "input/output are only allowed within IO block"
+      }
+    ]
+  }
+]
+ */
+package test
+
+object DisableRegex {
+  import scala.collection.mutable._ // ok
+  import scala.collection.mutable   // ok
+
+  @SuppressWarnings(Array("Disable.ListBuffer"))
+  @SuppressWarnings(Array("Disable.<init>"))
```

Comment:
I think we should not report duplicate errors on `new X` if `X` is disabled. I think it would be better to skip `<init>` because there is no way to target a constructor without referencing the explicit type.

Not sure what to do if the user wants to disable a specific constructor overload.

Reply:
In this case I'm disabling regex `"scala.collection.mutable.*"`. I'm not sure there's a good solution to ignore `<init>`  in regexes. I'd say regex initially is kind of hack, everyone who uses it should expect it's not a silver bullet.

Reply:
a good usecase is to allow creating `File`, but not doing anything with it. I'm fine with this meaning that overloads are treated equally. I custom syntax in the future might be nice, but it's a second order feature for sure.